trie/zk/merkle_tree_test.go

```diff
@@ -163,6 +163,38 @@ func TestProof(t *testing.T) {
 	}
 }
 
+func TestCanonicalValue(t *testing.T) {
```

Comment:
`TestCanonicalValueWithKeyPreimage` isn't better?

Reply:
I've reflected it.